OpenQA/Isotovideo/Utils.pm

```diff
@@ -53,6 +55,33 @@ sub git_remote_url ($git_repo_dir) {
     return 'UNKNOWN (error on git remote call)';    # uncoverable statement
 }
 
+sub _handle_caching ($clone_url, $clone_depth, $branch, $clone_cmd, $handle_output) {
+    # determine cache directory and ensure its parent directory exists
+    return undef unless my $git_cache_dir = $bmwqemu::vars{GIT_CACHE_DIR};
```

Comment:
Can you create a documentation string in doc/backend_vars.asciidoc? IMHO this is often the best approach to even start with that before writing the test, i.e. "README Driven Development" :)